src/components/DayPicker.jsx

```diff
@@ -28,6 +28,7 @@ const propTypes = {
   withPortal: PropTypes.bool,
   hidden: PropTypes.bool,
   initialVisibleMonth: PropTypes.func,
+  scrollable: PropTypes.bool,
```

Comment:
To enforce the vertical requirement, I can add a new orientation instead of using this additional prop.